keyboards/keebio/levinson/keymaps/issmirnov/ascii-keymap.txt

```diff
@@ -0,0 +1,55 @@
+                                   Qwerty                                    
```

Comment:
Please remove these text files, or put them into the readme.

Reply:
Done.